app/code/Magento/InventoryExportStock/Model/ResourceModel/StockIndexDumpProcessor.php

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryExportStock\Model\ResourceModel;
+
+use Exception;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Adapter\AdapterInterface;
+use Magento\Framework\DB\Select;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\InventoryExportStock\Model\GetQtyForNotManageStock;
+use Magento\InventoryIndexer\Model\StockIndexTableNameResolverInterface;
+use Magento\InventorySales\Model\ResourceModel\IsStockItemSalableCondition\ManageStockCondition as NotManageStockCondition;
+use Psr\Log\LoggerInterface;
+use Zend_Db_Expr;
```

Comment:
let's not use it at all

Reply:
We can't avoid it. We use Zend_DB_select. It doesn't work with Zend\Db\Sql\Expression

Reply:
ok, makes sense